pkg/printers/internalversion/printers.go

```diff
@@ -715,7 +715,12 @@ func getServiceExternalIP(svc *api.Service, wide bool) string {
 	case api.ServiceTypeLoadBalancer:
 		lbIps := loadBalancerStatusStringer(svc.Status.LoadBalancer, wide)
 		if len(svc.Spec.ExternalIPs) > 0 {
-			result := append(strings.Split(lbIps, ","), svc.Spec.ExternalIPs...)
+			var result []string
```

Comment:
Thank for your review very much. The old code will add a incorrect comma into output. My change can fix it.